pkg/doc.txt

```diff
@@ -14,14 +14,23 @@
  * INCLUDE += $(RIOTPKG)/<pkg_name>/...
  * ~~~~~~~~
  *
- * When the package can be build out-of-source, the source code of external
- * packages is fetched in a global location in
- * `$(RIOTBASE)/build/pkg/$(PKG_NAME)`.
+ * When the package can be built out-of-source, the source code of external
+ * packages is fetched in a global location in:
+ *
+ * ~~~~~~~~ {.mk}
+ * $(RIOTBASE)/build/pkg/$(PKG_NAME)
```

Comment:
Any particular reason make syntax is used here?

Reply:
Yes, it is make code :)

Reply:
Ok. From I thought it was just a location in which case I would have preferred shell-style variable syntax

```suggestion
 * ${RIOTBASE}/build/pkg/${PKG_NAME}
```

Reply:
I disagree, the `PKG_NAME` is only available in Make. This is only a build system thing.

Reply:
Also note that the code already provided above this additional note is also given in make code.